cristin-person/src/main/java/no/unit/nva/cristin/person/model/nva/Affiliation.java

```diff
@@ -0,0 +1,108 @@
+package no.unit.nva.cristin.person.model.nva;
+
+import static nva.commons.core.attempt.Try.attempt;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import java.net.URI;
+import java.util.Objects;
+import no.unit.nva.cristin.person.model.cristin.CristinAffiliation;
+import nva.commons.core.JacocoGenerated;
+
+@JacocoGenerated
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
+public class Affiliation {
+
+    private final URI organization;
+    private final Boolean active;
```

Comment:
Boolean can be null, what is the meaning of null?

Reply:
You want me to use primitive instead?

Reply:
yes